server/views/partials/header.njk

```diff
@@ -17,8 +17,8 @@
       <ul class="hmpps-header__navigation">
         {% if caseLoads.length > 1 %}
           <li class="hmpps-header__navigation__item">
-            <img class="hmpps-header__link__icon" src="/assets/images/location.svg" aria-hidden="true" alt="" />
-            <a data-qa="changeCaseLoad" class="hmpps-header__link" href="{{ changeCaseLoadLink }}" data-test="manage-account-link" target="_blank" rel="noopener noreferrer">
+            <img class="hmpps-header__link__icon govuk-!-padding-right-2" src="/assets/images/location.svg" aria-hidden="true" alt="" />
+            <a data-qa="changeCaseLoad" class="hmpps-header__link" href="{{ changeCaseLoadLink }}" data-test="change-case-load-link" target="_blank" rel="noopener no-referrer-when-downgrade">
```

Comment:
`rel` only supports `noreferrer` - need to use `referrerpolicy="no-referrer-when-downgrade"` instead

(Can keep `rel="noopener"` though)

Reply:
ah, will sort. Thanks